service_capacity_modeling/interface.py

```diff
@@ -74,6 +74,38 @@ def __hash__(self):
     def __eq__(self, other):
         return self.__hash__() == other.__hash__()
 
+    def scale(self, factor: float):
+        minimum_value = (
+            self.minimum * factor if self.minimum_value is not None else None
+        )
+        maximum_value = (
+            self.maximum * factor if self.maximum_value is not None else None
+        )
+        return Interval(
+            low=self.low * factor,
+            mid=self.mid * factor,
+            high=self.high * factor,
+            confidence=self.confidence,
+            model_with=self.model_with,
+            allow_simulate=self.allow_simulate,
+            minimum_value=minimum_value,
+            maximum_value=maximum_value,
+        )
+
+    def offset(self, delta):
```

Comment:
nit: type annotation
```
def scale(self, delta: float) -> "Interval":
```

Reply:
i had it them in there at one point but was playing with the idea implementing them as operator overloads. It was way more code and figured you wouldn't like it and switched back.